mail/opendkim/files/opendkim-genkey

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+if [ -x /usr/bin/perl ]; then
+  /usr/sbin/opendkim-genkey.pl
+else
+  echo ""
+  echo "Perl is required for running this script. Install Perl or run"
+  echo "this script on another computer that has Perl with File::Basename"
+  echo "and Getopt::Long modules installed."
+  echo ""
+fi
```

Comment:
Yes, that's correct. This Perl script is needed only once for the generation of keys. It can be executed on a computer that has Perl installed. Making this package depend on Perl so that this script can be executed on an embedded device is an overkill IMHO.

If Perl is already installed on the device, opendkim-genkey executes normally.